tests/test_parse_requirements.py

```diff
@@ -235,10 +237,14 @@ def test_parse_requirements_editable_escaped_path(monkeypatch, data):
 
 
 def test_parse_requirements_with_relative_references(monkeypatch):
+    # NOTE: The top-level file is accessed via the literal path passed into
+    # `parse_requirements`, while relative files are accessed by joining them
+    # with the directory. Hence the top-level has a forward slash always,
+    # while the relative files have whatever path separator the host uses.
     files = {
-        "reqs/base.txt": ["django==1.11\n"],
-        "reqs/test.txt": ["-r base.txt\n"],
-        "reqs/dev.txt": ["-r base.txt\n" "-r test.txt\n"],
+        os.path.join("reqs", "base.txt"): ["django==1.11\n"],
+        os.path.join("reqs", "test.txt"): ["-r base.txt\n"],
+        "reqs/dev.txt": ["-r base.txt\n-r test.txt\n"],
     }
```

Comment:
This is a bit hacky, but I figured it was better to update the unit tests to match the observed behavior (paths are sometimes native, sometimes not) rather than attempt a larger refactor within this specific PR 🙂